src/_nebari/upgrade.py

```diff
@@ -141,12 +142,10 @@ def upgrade_step(self, config, start_version, config_filename, *args, **kwargs):
         for any actions that are only required for the particular upgrade you are creating.
         """
         finish_version = self.get_version()
-        __rounded_finish_version__ = ".".join(
-            [str(c) for c in rounded_ver_parse(finish_version)]
-        )
+        ".".join([str(c) for c in rounded_ver_parse(finish_version)])
```

Comment:
` __rounded_finish_version__` was this variable dropped?

Reply:
Yeah I should be using this... I added it back.